kubernetes/launch-job.sh

```diff
@@ -187,8 +209,46 @@ else
   CURRICULUM_CMD="/go_attack/kubernetes/curriculum.sh $RUN_NAME $VOLUME_NAME $CURRICULUM"
 fi
 
+if [ -n "${ONLY_LAUNCH_CURRICULUM:-}" ]; then
```

Comment:
is this mainly for local (non-hofvarpnir) runs? on hofvarpnir you'd have to kill the old curriculum, but i'm not certain if there's a way to kill just the curriculum pod without killing the whole `go-train-"$1"-vital` job

Reply:
Hmm, possibly. This was a while back and I forget if I used this on hofvarpnir or not. Do you know how you would in principle hot-swap the curriculum on hofvarpnir?